### ISSUE TITLE
Control cookies directly from Cookie Analysis extension

### ISSUE THREAD

**Issue by amedina** [GoogleChromeLabs/cookie-analysis-tool#20]
**Description:**
Facilitate testing of different scenarios by allowing users to control (e.g. block/unblock/configure) cookies directly from the extension. 

**Acceptance Criteria:**
- [ ] Block cookies
- [ ] Delete cookies

**Comment by amedina** [comment]
The idea is to capture the functionality available in Chrome's settings:

<img width="1971" alt="Screenshot 2023-06-13 at 6 37 49 PM" src="https://github.com/GoogleChromeLabs/cookie-analysis-tool/assets/506089/f9bbc037-ab2c-45a6-81ab-5e3b2dd4bba9">
<img width="1964" alt="Screenshot 2023-06-13 at 6 38 14 PM" src="https://github.com/GoogleChromeLabs/cookie-analysis-tool/assets/506089/00050825-f9ab-4fff-8f44-6964548f805d">

**Comment by amedina** [comment]
This feature for 0.4 should capture the semantics of the Cookies section in the Applications tab: 

<img width="349" alt="Screenshot 2023-11-07 at 1 31 57 PM" src="https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/506089/bce3699b-700d-439d-87a6-9365455626ee">

These two capabilities enable: (1) Clear all cookies; (2) Show only cookies with issues. 

cc/. @mohdsayed

**Comment by amovar18** [comment]
For more detailed technical implementation for this issue, you can check the PR description.

**Comment by prasadnevase** [comment]
Hi @amovar18 - great work. Few things we can amend:

- [ ] Sequence to resemble same as under Application tab: "Clear all cookies" then "Delete Selected" (Also note the title text)
- [ ] Shall we use the icons same as used in Application tab: "[Clear all cookies](https://github.com/ChromeDevTools/devtools-frontend/blob/main/front_end/Images/src/clear-list.svg)" and "[Delete Selected](https://github.com/ChromeDevTools/devtools-frontend/blob/main/front_end/Images/src/cross.svg)"
- [ ] As under Application tab, shall we show "Delete Selected" disabled by default (as none of the cookie is selected)
- [ ] "Only show cookies with an issue" - is missing. Are we going to consider this under a separate issue/request?
